### DISCUSSION TITLE
Delete multiple routes in one go

### DISCUSSION THREAD

**Discussion by angieherrera** [craftcms/cms#10069] [Ideas] [answered]
Would love to see a way to batch delete routes in the CP. Currently you can only delete one by one. When there are many, this takes more time than it should. 

For example, using the Calendar plugin, if you install the demo templates, it also installs several routes. This is great for testing and development, but once done with the demo stuff, it'd be great to select all the routes I want to delete and then click a Delete button (much like how you can delete multiple entries).

**Comment by brandonkelly** [comment, marked as answer]
Agree we should making deleting easier. Maybe even just an `x` button beside each route on the main listing view, which would at least save one click.

In the meantime, you might find it easier to delete routes directly from the project config. They are stored in `config/project/routes/`, and each route is a file in there named after the route’s UUID. (Realize that’s not a very human-friendly filename, but you could search the folder for `calendar` or whatever, to quickly find all the files that you’re looking to delete.)

After deleting them, run the following command to apply the changes:

```sh
php craft project-config/apply
```
